pkg/env/env.go

```diff
@@ -0,0 +1,19 @@
+package env
+
+import (
+	"os"
+
+	"github.com/spf13/viper"
+)
+
+const ()
```

Comment:
I'm not so good at Go

What is the usage of this line

Reply:
It is just I delete a value but not delete const(), I think maybe I will fill in some value oneday.